compiler/rustc_session/src/search_paths.rs

```diff
@@ -57,6 +63,27 @@ impl SearchPath {
             (PathKind::Framework, stripped)
         } else if let Some(stripped) = path.strip_prefix("all=") {
             (PathKind::All, stripped)
+        } else if let Some(stripped) = path.strip_prefix("builtin:") {
+            let Some(sysroot) = sysroot else {
+                early_dcx.early_fatal("`-L builtin:` is not supported without a sysroot present");
+            };
+            let triple = match triple {
```

Comment:
```suggestion
            let triple = triple.triple();
```
(That's how it's done in other places using `make_target_lib_path`.)